sdcm/provision/gce/__init__.py

```diff
@@ -0,0 +1,9 @@
+from typing import NamedTuple
+
+
+class GCERegions(NamedTuple):
+    us_east1: str = "us-east1"
+    us_west1: str = "us-west1"
```

Comment:
those are the only supported regions?
what is need to add more regions?

Reply:
Not sure if these are the only supported ones, but the only ones I've seen mentioned in our code. It's obviously simple to add more here, if we use them.